### ISSUE TITLE
Gnome 41 support

### ISSUE THREAD

**Issue by aravindhp** [JasonLG1979/gnome-shell-extension-weather-in-the-clock#24]
## Distro Name & Version
Fedora 35

## GNOME Shell Version
41.1

## Other Installed Extensions
```shell
  gnome-extensions list --enabled
appindicatorsupport@rgcjonas.gmail.com
trayIconsReloaded@selfmade.pl
user-theme@gnome-shell-extensions.gcampax.github.com
hibernate-status@dromi
```

## Expected Behavior
The extension should load without errors

## Current Behavior
The extension does not load

## Detailed Description
The extension does not load on Gnome 41.1 with the error `The extension is incompatible with the current GNOME version`

**Comment by aravindhp** [comment]
Might be fixed by https://github.com/JasonLG1979/gnome-shell-extension-weather-in-the-clock/commit/b36cc55bf41af1a4c801ddeb0fd71d1294892c4c

**Comment by JasonLG1979** [comment]
> Might be fixed by [b36cc55](https://github.com/JasonLG1979/gnome-shell-extension-weather-in-the-clock/commit/b36cc55bf41af1a4c801ddeb0fd71d1294892c4c)

I haven't uploaded the update yet I'll do that now.
